src/utils/Codec.ts

```diff
@@ -67,19 +67,19 @@ type StructType =
 
 // Enum for better autocomplete and type safety
 export enum Type {
-    Block,
-    BlockCommitment,
-    JoinChannel,
-    OpenChannel,
-    BlockConfirmation,
-    Transaction,
-    Dispute,
-    StateSnapshot,
-    SnapshotData,
-    JoinChannelBlock,
-    ExitChannelBlock,
-    ExitChannel,
-    DisputeAuditingData
+    Block = "Block",
+    BlockCommitment = "BlockCommitment",
+    JoinChannel = "JoinChannel",
+    OpenChannel = "OpenChannel",
+    BlockConfirmation = "BlockConfirmation",
+    Transaction = "Transaction",
+    Dispute = "Dispute",
+    StateSnapshot = "StateSnapshot",
+    SnapshotData = "SnapshotData",
+    JoinChannelBlock = "JoinChannelBlock",
+    ExitChannelBlock = "ExitChannelBlock",
+    ExitChannel = "ExitChannel",
+    DisputeAuditingData = "DisputeAuditingData"
```

Comment:
I assume the `strings` are added for debugging purposases and are not the cause of the error / don't make a logical difference?

Reply:
Let's have a call. The 100 was the source of the bug and the strings are part of the solution